WordPressKit.podspec

```diff
@@ -18,9 +18,10 @@ Pod::Spec.new do |s|
   s.swift_version = '5.0'
 
   s.source        = { git: 'https://github.com/wordpress-mobile/WordPressKit-iOS.git', tag: s.version.to_s }
-  s.source_files  = 'WordPressKit/**/*.{h,m,swift}'
-  s.private_header_files = 'WordPressKit/Private/*.h'
-  s.header_dir = 'WordPressKit'
+  s.source_files  = 'Sources/**/*.{h,m,swift}'
```

Comment:
I could have made this `Sources/WordPressKit/*.{h,m,swift}` but I thought I'd leave it looser for the moment. I'm not sure how/if to sort how subspecs as well as subpackages.